<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>fm_nova_venda</class>
 <widget class="QDialog" name="fm_nova_venda">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>752</width>
    <height>514</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>420</y>
     <width>81</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Valor total:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_total_venda">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>420</y>
     <width>111</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>R$ 0,00</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_fechar">
   <property name="geometry">
    <rect>
     <x>666</x>
     <y>483</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Fechar</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget_itens_venda">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>731</width>
     <height>311</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>6</x>
     <y>3</y>
     <width>521</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Quantidade:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_quantidade_produto">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>44</y>
      <width>21</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_id_produto">
    <property name="geometry">
     <rect>
      <x>176</x>
      <y>40</y>
      <width>181</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>175</x>
      <y>20</y>
      <width>141</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>CÃ³digo do produto:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_pesquisar_produto">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>40</y>
      <width>130</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Pesquisar produto</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>451</y>
     <width>381</width>
     <height>27</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="pushButton_excluir_item">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Excluir produto</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_editar_item">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Editar produto</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_finalizar_venda">
      <property name="text">
       <string>Finalizar venda</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>420</y>
     <width>121</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Forma_pagamento:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_forma_pagamento">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>420</y>
     <width>301</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>lineEdit_quantidade_produto</tabstop>
  <tabstop>lineEdit_id_produto</tabstop>
  <tabstop>pushButton_pesquisar_produto</tabstop>
  <tabstop>tableWidget_itens_venda</tabstop>
  <tabstop>pushButton_excluir_item</tabstop>
  <tabstop>pushButton_editar_item</tabstop>
  <tabstop>pushButton_finalizar_venda</tabstop>
  <tabstop>pushButton_fechar</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
